src/components/NodeSelectorPanel/RepresentativeNodeWrapper.tsx

```diff
@@ -18,10 +19,9 @@ interface RepresentativeNodeWrapperProps {
 }
 
 const RepresentativeNodeWrapper = ({ node }: RepresentativeNodeWrapperProps) => {
-    const { createNode, reactFlowInstance, reactFlowWrapper, useHoveredNode } =
-        useContext(GlobalContext);
-
-    const [, setHoveredNode] = useHoveredNode;
+    const createNode = useContextSelector(GlobalChainContext, (c) => c.createNode);
+    const reactFlowInstance = useContextSelector(GlobalChainContext, (c) => c.reactFlowInstance);
```

Comment:
Didn't we get rid of reactFlowInstance? Is this still needed?

Reply:
After doing a code search I've determined that reactFlowInstance is not needed and can be removed.

Reply:
There are two instances where we use `reactFlowInstance.project()`. How would those be replaced?

Reply:
That can be replaced with the new (in v10) useReactFlow hook which contains the project method https://reactflow.dev/docs/api/hooks/use-react-flow/ https://reactflow.dev/docs/api/react-flow-instance/

Reply:
Good to know.